src/SegmentedArray.chpl

```diff
@@ -40,6 +40,13 @@ module SegmentedArray {
         return getSegArray(name, st, segEntry.etype);
     }
 
+    proc assembleSegArrayFromParts(segments:SymEntry, values:SymEntry, st:borrowed SymTab, type eltType): owned SegArray throws {
```

Comment:
Based on updates requested in `HDF5Msg.chpl`, this can be removed.

Reply:
Removed